### ISSUE TITLE
--noconfirm skips AUR package selection

### ISSUE THREAD

**Issue by SicariusNoctis** [trizen/trizen#153]
From a user perspective, the spirit of the `--noconfirm` option is to skip through the countless mundane "enter" presses that occur at arbitrary times during a full system upgrade `-Syyu`.

However, the package selection screen is rather important and happens only once. I don't think it should be skipped. This issue causes me to revert to other package managers when I require AUR package selection. 

I think the AUR package selection screen should not be skipped. And should occur very early on, immediately after synchronizing the package databases (`-yy`).

**Comment by mbloms** [comment]
The purpose of noconfirm is automation, not to allow users to be lazy.
lör 21 apr. 2018 kl. 06:24 skrev Mateen Ulhaq <notifications@github.com>:

**Comment by SicariusNoctis** [comment]
What's the difference between a user being lazy and automation?

**Comment by trizen** [comment]
@SicariusNoctis maybe `--noedit` is what you are looking for?

**Comment by SicariusNoctis** [comment]
Almost. It does allow me to select the right AUR packages. But also it requires me to confirm each AUR package install. (Which isn't very useful since I've already said I wanted those specific packages!)

**Comment by trizen** [comment]
Please take a look at the config option: `install_built_with_noconfirm`.

**Comment by SicariusNoctis** [comment]
Thanks, this seems to do the trick:

    trizen -Syu --install_built_with_noconfirm --noedit

Does it still give prompts for conflicts? (I think those are still important enough to prompt the user.)

**Comment by trizen** [comment]
For conflicts and replacements, I think you may need to add the `--ask` argument to the `pacman_command`.

**Comment by trizen** [comment]
No longer relevant, as `install_built_with_noconfirm` has been removed. (https://github.com/trizen/trizen/commit/c593403e6951c1a696c210cc1654e2c779cffb88)
